### PR TITLE
Added logsoftmax operator

### LINKED ISSUE #128: Implement LogSoftmax operator
https://github.com/onnx/onnx/blob/main/docs/Operators.md#LogSoftmax

### LINKED ISSUE #207: Missing tests for Less, LessOrEqual and LinearRegressor in opset13_test

### PR DESCRIPTION
**Pull request by Swopper050**
Closes #128, Closes #207

First merge #179 
Probably better to first implement ReduceMax before merging this one. This is needed for some of the ONNX tests

### CODE REVIEW

File: ops/opset13/opset13_test.go [hunk 1]
```diff
@@ -133,6 +133,26 @@ func TestGetOperator(t *testing.T) {
...
+			"LinearRegressor",
+			newLinearRegressor(),
+			nil,
+		},
+		{
+			"LogSoftmax",
+			newLogSoftmax(),
+			nil,
+		},
 		{
```
Comment (wipsel) [inline ops/opset13/opset13_test.go:156]: Why these changes?

**Review by wipsel — CHANGES_REQUESTED** [review]

File: ops/opset13/opset13_test.go (hunk 1 above)
Comment (Swopper050) [reply ops/opset13/opset13_test.go:156]: These operators were not added to the tests in earlier pull request.

Comment (wipsel) [reply ops/opset13/opset13_test.go:156]: Maybe nice to create an issue for it and mention it in this pull request so it is clear in the hit history that this also changed?

**Review by wipsel — APPROVED** [review]
